src/main.ts

```diff
@@ -37,6 +40,40 @@ async function bootstrap(): Promise<void> {
     }),
   );
   app.setGlobalPrefix('api');
+
+  const options = new DocumentBuilder()
+    .setTitle('Keyman Developer Online API')
+    .setDescription(`<p>Contribute keyboards to the Keyman keyboards repo. The API allows to
+convert a single keyboard repo into a pull request on the huge
+<a href="https://github.com/keymanapp/keyboards">keyboards</a> repo.</p>
+<p>The OpenAPI JSON file can be downloaded <a href="/api-json">here</a>.</p>`)
+    .setVersion('0.1')
+    .addTag('user related')
+    .addTag('project related')
+    .addOAuth2(
+      // NOTE: Currently it's not possible to login through Swagger UI using this kind of
+      // authentication. I guess I'm missing something, but I cannot figure out what...
+    {
+      type: 'oauth2',
+      description: 'just testing',
+      flows: {
+        authorizationCode: {
+          authorizationUrl: 'https://github.com/login/oauth/authorize',
+          tokenUrl: 'https://github.com/login/oauth/access_token',
+          scopes: GithubService.scope.split(' '),
+        }
+      }
+    }, 'Default')
+    .addBasicAuth({
+      type: 'http',
+      description: 'Used only in e2e tests. Use GitHub <b>username</b> and <b>personal access token</b>.',
+    }, 'Tests')
+    .addServer('http://localhost:3000', 'Local development server')
+    .addServer('https://keyman-developer-online.azurewebsites.net', 'Main production server')
```

Comment:
I presume we'll need to tweak this for the future -- I'd like to see this surfaced as a global property somewhere so it is mentioned only once.

Reply:
I removed the two `.addServer` lines - we don't gain much and they are not needed for Swagger UI to work.